scripts/submit_emod3d.py

```diff
@@ -109,11 +117,8 @@ def main(args):
         binary_path = binary_version.get_lf_bin(
             params.emod3d.emod3d_version, target_qconfig["tools_dir"]
         )
-
         steps_per_checkpoint = int(
-            get_nt(params)
-            / (60.0 * convert_time_to_hours(est_run_time))
```

Comment:
Any reason for removing the convert_time_to_hours call?

Reply:
est_run_time already in hours. Using covert_time_to_hours results in run time error

Reply:
:+1: